src/platform/user/authentication/components/LoginActions.jsx

```diff
@@ -11,8 +11,11 @@ export default ({
   loginGovMyVAHealthEnabled,
 }) => {
   const externalLoginGovSupport = {
-    mhv: loginGovMHVEnabled,
-    myvahealth: loginGovMyVAHealthEnabled,
+    [EXTERNAL_APPS.MHV]: loginGovMHVEnabled,
+    [EXTERNAL_APPS.MY_VA_HEALTH]: loginGovMyVAHealthEnabled,
+    [EXTERNAL_APPS.VA_FLAGSHIP_MOBILE]: loginGovEnabled,
+    [EXTERNAL_APPS.VA_OCC_MOBILE]: loginGovEnabled,
+    [EXTERNAL_APPS.EBENEFITS]: loginGovEnabled,
```

Comment:
@AGarcia-Clarity Added ebenefits here as it should also enable/disable login.gov based on this flag

Reply:
When we discussed previously, I thought only My VA Health (Cerner) and MHV should have the loginGovEnabled and the rest would go with our normal flow.

Reply:
What I got from that conversation was that Ebenefits/OCC/Flagship would not need their own flags.

MHV uses `loginGovMHVEnabled`
MyVAHealth uses `loginGovMyVAHealthEnabled`

The new 3 still need to know if the global `loginGovEnabled` is `true` or they would never display.